src/escsm.js

```diff
@@ -27,11 +27,9 @@
  * 02110-1301  USA
  */
 
-!function(jschardet) {
+var consts = require('./constants');
 
-var consts = jschardet.Constants;
-
-jschardet.HZ_cls = [
+exports.HZ_cls = [
```

Comment:
´HZ_cls` and a few more are never used outside of this scope so we don’t really need to add it to `exports`. Not sure why I added it to `jschardet` at the time, but we can avoid it now.